packages/flutter/lib/src/widgets/basic.dart

```diff
@@ -333,7 +333,75 @@ class ShaderMask extends SingleChildRenderObjectWidget {
   }
 }
 
-/// A widget that applies a filter to the existing painted content and then paints [child].
+/// A widget that applies a filter to the existing painted content and then
```

Comment:
Maybe "to the counted already painted within its bounds"?

Reply:
It seems to happily apply the filter outside its bounds if a child decided to paint outside the bounds...